File: ClimaTempo.Web/Entidades/Locale.cs
using Newtonsoft.Json;
using System;


namespace ClimaTempo.Web.Entidades
{
    public class Locale
    {

        [JsonProperty("id")]
        public int id { get; set; }
        [JsonProperty("name")]
        public string name { get; set; }
        [JsonProperty("state")]
        public string state { get; set; }
        [JsonProperty("latitude")]
        public double latitude { get; set; }
        [JsonProperty("longitude")]
        public double longitude { get; set; }
    }
}

File: packages/api/src/entities/Locale.ts
import { v4 as uuidv4 } from 'uuid';

export class Locale {
  public readonly id: string;
  name: string;
  state: string;
  latitude: number;
  longitude: number;

  constructor(props: Omit<Locale, 'id'>, id?: string) {
    Object.assign(this, props);

    if (!id) this.id = uuidv4();
  }
}
